Adding disclaimer to Auth0 migration (#5655)
fishtown-analytics/docs.getdbt.com: website/docs/docs/cloud/manage-access/auth0-migration.md

description: "Required actions for migrating to Auth0 for SSO services on dbt Cloud."
---

:::note

> This guide describes a feature of the dbt Cloud Enterprise plan. If you want to learn more about an Enterprise plan, contact us at [sales@getdbt.com](mailto::sales@getdbt.com).

Reviewer: Using "guide" might be misleading because we have a content type of guides. Elsewhere I see we sometimes say "This feature is limited to the dbt Cloud Enterprise plan." or "People on Enterprise plans can... " 

We should probably get these all to align. I kind of prefer focusing on the features like "Auth0 for SSO is limited to the dbt Cloud Enterprise plan" because that feels clear, but happy to go with any variation that makes most sense to everyone. 

```suggestion
Auth0 for SSO is limited to the dbt Cloud Enterprise plan. If you want to learn more about an Enterprise plan, contact us at [sales@getdbt.com](mailto::sales@getdbt.com).
```

Author: That makes sense, but this is specifically a migration feature. How about:
```suggestion
This migration is a feature of the dbt Cloud Enterprise plan. To learn more about an Enterprise plan, contact us at [sales@getdbt.com](mailto::sales@getdbt.com).
```

Reviewer: AH got it! 👍🏻

> This guide describes a feature of the dbt Cloud Enterprise plan. If you’re interested in learning more about an Enterprise plan, contact us at [sales@getdbt.com](mailto::sales@getdbt.com).

Author: ```suggestion
This guide describes a feature of the dbt Cloud Enterprise plan. If you want to learn more about an Enterprise plan, contact us at [sales@getdbt.com](mailto::sales@getdbt.com).
```

> Single-tenant Virtual Private users should [email dbt Cloud Support](mailto::support@getdbt.com) to set up or update their SSO configuration.

Reviewer: We should probably make this consistently 2nd person:
```suggestion
For single-tenant Virtual Private Cloud, you should [email dbt Cloud Support](mailto::support@getdbt.com) to set up or update your SSO configuration.
```

> The SSO migration instructions apply to multi-tenant Enterprise deployments only. Single-tenant Virtual Private users should [email dbt Cloud Support](mailto::support@getdbt.com) to set up or update their SSO configuration.

Author: ```suggestion
Single-tenant Virtual Private users should [email dbt Cloud Support](mailto::support@getdbt.com) to set up or update their SSO configuration.
```
